scripts/download_gherkin_features.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -ex
+
+download_gherkin()
+{
+    rm -rf ${1} && mkdir -p ${1}
+    for run in 1 2 3 4 5
+    do
+        if [ -x "$(command -v gtar)" ]; then
+            curl --silent --fail https://codeload.github.com/elastic/apm/tar.gz/${2} | gtar xzvf - --directory=${1} --strip-components=4 "*/tests/agents/gherkin-specs*"
+        else
+            curl --silent --fail https://codeload.github.com/elastic/apm/tar.gz/${2} | tar xzvf - --directory=${1} --strip-components=4 "*tests/agents/gherkin-specs*"
```

Comment:
@beniwohli I removed the `--wildcard` flag to make these commands cross-platform